sdk/appconfiguration/app-configuration/src/internal/helpers.ts

```diff
@@ -14,7 +14,7 @@ import {
   ConfigurationSnapshot,
   SnapshotResponse,
   EtagEntity,
-} from "../models";
+} from "../models/index.js";
```

Comment:
Why is this one using index.js and the rest aren't?

Reply:
I was initially tweaking `app-config` but then switched to tweaking `template` instead, so the PR doesn't bloat up.
This was one of the leftovers, removed now.